src/views/SpaceProposal.vue

```diff
@@ -120,6 +136,19 @@ async function loadResults() {
   }
 }
 
+const { loadBy, loadingMore, stopLoadingMore, loadMore } =
+  useInfiniteLoader(10);
+
+async function loadMoreVotes() {
+  console.log('loadMORE');
```

Comment:
If this console log is still here on purpose, it could include the params.

Reply:
Nah that snuck in somehow 😆